build.gradle

```diff
@@ -167,15 +167,11 @@ bootRun {
     jvmArgs = [
             "-DnativeDir=build/resources/native",
             "-Dprism.forceGPU=true",
-            "-Xms128m",
-            "-Xmx712m",
+            "-Xmx1g",
```

Comment:
Why?

Reply:
With how ZGC operates it needs a little bit more heap overhead in order to operate properly without constantly performing concurrent GCs and kicking up CPU usage